src/material/tree/testing/tree-harness.ts

```diff
@@ -28,4 +28,17 @@ export class MatTreeHarness extends ComponentHarness {
   async getNodes(filter: TreeNodeHarnessFilters = {}): Promise<MatTreeNodeHarness[]> {
     return this.locatorForAll(MatTreeNodeHarness.with(filter))();
   }
+
+  async getTreeStructure(): Promise<string> {
```

Comment:
Hmm how about calling it `getStructuredText`? and include a doc comment explaining what that structure is